CONTRIBUTING.md

```diff
@@ -21,18 +21,18 @@ key, but non-obvious, aspects:
 - All descriptions, normative language are defined in the `docs/`
   directory.
   - We provide tooling to generate Markdown documentation from the formal
-    YAML definitons.  See [Yaml to Markdown](#yaml-to-markdown).
+    YAML definitons. See [Yaml to Markdown](#yaml-to-markdown).
```

Comment:
Changes in this file seem to be unrelated?

Reply:
That's true. These changes are caused by `make fix` suggested [in contributing.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#autoformatting). 

Should I create a separate PR for that?

Reply:
Ah ok! I guess we are good to add those here since they are minor unless we have any objection from someone else.

Reply:
> unless we have any objection from someone else.

I hope not :) because anyone who does `make fix` in any upcoming PRs would make the same changes.